### PR TITLE
Maintain a signle graph database instance per metrics-exporter

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
Haven't tested yet, not sure if the connection pool will transparently recover from failure (worth some tests).

File: thoth/metrics_exporter/jobs/inspection.py [hunk 1]
```diff
@@ -19,7 +19,6 @@
 
 import logging
 
-from thoth.storages import GraphDatabase
```
Comment (pacospace) [inline thoth/metrics_exporter/jobs/inspection.py]: Also `get_inspection_results_per_identifier` is a class method

Comment (fridex) [reply thoth/metrics_exporter/jobs/inspection.py]: It's a static method in sources - it does not need graph database instance. Or am I missing something?

Comment (pacospace) [reply thoth/metrics_exporter/jobs/inspection.py]: All good thanks :)

**Pull request merged**